ta_services/keymaster/km.c

```diff
@@ -0,0 +1,12 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
```

Comment:
minor: SPDX format
> (Rule 1.2.1) C source: // SPDX-License-Identifier: <expression>

Reply:
Good catch, let's take that in next patch now that it's merged.